e2e/cypress/support/api/user.js

```diff
@@ -256,3 +256,10 @@ Cypress.Commands.add('apiPromoteGuestToUser', (userId) => {
         return cy.apiGetUserById(userId);
     });
 });
+
+Cypress.Commands.add('apiGetPreferences', (userId) => {
```

Comment:
I don't see this used elsewhere but may help in the future. So, I'm fine to include as is. However please move into `support/api/preference.js` as we're currently arranging API commands and would like to match the structure per [API reference](https://api.mattermost.com/). Also, please add type at `preference.d.ts`.

Reply:
originally I was trying to get if the new sidebar was active to choose one or the other, but ended up rejecting the idea as it was getting complicated to detect and I thought it would be better to specify in the test if that was desired.

But as you, I thought it might come useful in the future and didn't delete it